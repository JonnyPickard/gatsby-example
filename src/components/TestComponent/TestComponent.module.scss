.testComponent {
  background: lightgreen;
  width: 100vh;
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: 'Courier New', Courier, monospace;
  color: white;
  font-size: 40px;
}