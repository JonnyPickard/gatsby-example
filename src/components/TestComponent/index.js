import React from 'react'
import styles from './TestComponent.module.scss'

class TestComponent extends React.Component {
  render() {
    return <div className={styles.testComponent}>Hello from TestComponent</div>
  }
}

export default TestComponent
