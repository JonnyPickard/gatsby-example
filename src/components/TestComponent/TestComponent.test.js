import React from 'react'
import Enzyme, { mount } from 'enzyme'
import Adapter from 'enzyme-adapter-react-15'
import TestComponent from '.'

Enzyme.configure({
  adapter: new Adapter(),
})

describe('TestComponent', () => {
  it('renders TestComponent', () => {
    const wrapper = mount(<TestComponent />)
    expect(wrapper).toMatchSnapshot()
  })
})
