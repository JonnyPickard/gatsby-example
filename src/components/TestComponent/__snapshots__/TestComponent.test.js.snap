// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent renders TestComponent 1`] = `
<TestComponent>
  <div
    className="testComponent"
  >
    Hello
  </div>
</TestComponent>
`;
